.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.lock
 behat.yml
 *.tar.gz
+/.idea/
```

Comment:
It's OK to add this, but if we are touching this file and are introducing the (better) practice of prefixing the paths with a slash to capture the exact location in the tree structure, let's then also do this for the existing paths, so we are consistent.